spec/migrations/20180424141617_azure_backslash_to_forward_slash_spec.rb

```diff
@@ -0,0 +1,86 @@
+require_migration
+
+describe AzureBackslashToForwardSlash do
+  let(:vm_stub) { migration_stub :Vm }
+  let(:stack_stub) { migration_stub :OrchestrationStack }
+  let(:stack_output_stub) { migration_stub :OrchestrationStackOutput }
+
+  let!(:vm) do
+    vm_stub.create!(
+      :name        => 'azure_test_vm',
+      :ems_ref     => 'xyz\some_group\microsoft.compute/virtualmachines\foo',
+      :type        => 'ManageIQ::Providers::Azure::CloudManager::Vm',
+      :description => 'some\description'
+    )
+  end
+
+  let!(:template) do
+    vm_stub.create!(
+      :name        => 'azure_test_template',
+      :ems_ref     => 'xyz\some_group\microsoft.compute/virtualmachines\bar',
+      :type        => 'ManageIQ::Providers::Azure::CloudManager::Template',
+      :description => 'some\other\description'
+    )
+  end
+
+  let!(:stack) do
+    stack_stub.create!(
+      :id      => 1,
+      :name    => 'azure_test_stack',
+      :ems_ref => 'xyz/resourceGroups\foo\providers/Microsoft.Resources/deployments/stuff',
+      :type    => 'ManageIQ::Providers::Azure::CloudManager::OrchestrationStack'
+    )
+  end
+
+  let!(:stack_output) do
+    stack_output_stub.create!(
+      :stack_id => 1,
+      :key      => 'adminUserName',
+      :value    => 'me',
+      :ems_ref  => '/subscriptions/xyz/resourceGroups/some_group/providers/Microsoft.Resources/deployments/RedHat\adminUsername'
+    )
+  end
+
+  let!(:stack_param) do
+    stack_param.create!(
```

Comment:
@djberg96 This is a infinite recursion loop here.  Is it possible that this is causing your core dump?

Reply:
I changed this to `stack_output_stub.create!` instead of `stack_param.create!`, and it seemed to allow the tests to pass.

Reply:
Yep, that was it. Will update the stub, thanks.